server/mergin/sync/public_api.yaml

```diff
@@ -299,22 +300,22 @@ paths:
           $ref: "#/components/responses/NotFoundResp"
       x-openapi-router-controller: mergin.sync.public_api_controller
   /project/by_uuid/{project_id}:
```

Comment:
it is actually a good opportunity to start using `/v2` version as we will need to rewrite all project-like endpoints to support project id instead of namespace+name